app/containers/Quiz/index.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from 'react';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+
+import * as quizActionCreators from 'actions/quiz';
+
+const assembledActionCreators = Object.assign({}, quizActionCreators);
+
+import Question from 'components/Question';
+
+import {styles} from './styles.scss';
+
+const mapStateToProps = state => {
+  return {
+    question: state.quiz.question
+  };
+}
+
+@connect (
```

Comment:
they are certainly not standard (yet) but they are too powerful and fun to play around with to go back to the dirty way IMO. i would argue to leave them in for now and if they prove troublesome or confusing we can revert.